pkg/runtest/run_test.go

```diff
@@ -21,7 +21,11 @@ func Test(t *testing.T) {
 	case "openbsd":
 		t.Skipf("broken on %v", runtime.GOOS)
 	}
+	shortTarget := targets.Get("test", "64")
 	for _, sysTarget := range targets.List["test"] {
+		if testing.Short() && sysTarget != shortTarget {
```

Comment:
As a general note, one problem I still have looking at the syzkaller codebase is that there are hardly any comments. I know it's very likely you're the only one to ever look at this (except during review), but as some way to help out future people looking at this, it'd be very useful if the comment in the commit message could be moved into the code.

It's also hard to say for me what this file is meant to do. A top-level comment could be useful here.

Reply:
I have a problem calibrating what's obvious for other readers and what's not.
Say here: if there is an if on testing.Short in a test, that's almost always to reduce duration of the test in short mode because it takes too long (default rationale). We also run only 1 test target instead of all, so that's how we reduce duration in short mode.
If we agree on reviewing all changes and somebody will give me feedback, then I will calibrate better over time (also won't be able to cut corners :)).

I've added a comment here.

Reply:
Hard to say what's best.

I generally try to add comments summarizing large code blocks, functions, or the entire compilation unit, even if most of it is trivial.

It's like reading a book. I first want to have a summary that tells me if it's even worth reading the book (e.g. the back cover of the book, or its Wikipedia article), without having to read the whole book to figure out if I should have read it in the first place. ;-)

Reply:
I try to do this as well:
https://github.com/google/syzkaller/blob/master/pkg/runtest/run.go#L4-L11